dashboard/client/src/pages/serve/ServeApplicationDetailPage.tsx

```diff
@@ -138,6 +138,12 @@ export const ServeApplicationDetailPage = () => {
               />
             ),
           },
+          {
+            label: "Import path",
+            content: {
+              value: application?.deployed_app_config?.import_path || "",
```

Comment:
```suggestion
              value: application?.deployed_app_config?.import_path || "-",
```

This matches the pattern we use in other metadata fields.

Reply:
Good catch!